ggshield/cmd/sca/scan/ci.py

```diff
@@ -75,10 +76,7 @@ def scan_ci_cmd(
             ci_mode, wd=directory, verbose=config.user_config.verbose
         )
         if params is None:
-            click.echo(
-                "No commit found in merge request, skipping scan.",
-                err=True,
-            )
+            ui.display_info("No commit found in merge request, skipping scan.")
```

Comment:
This is a bit confusing: `click.echo("foo", err=True)` does not mean it's an error, it means: print to stderr instead of printing to stdout. We do this for all progress information. It is important to do this for all progress output because otherwise it breaks JSON to stdout.

Reply:
Oh I see, thanks :)